migration/1648066794387-addChangeAcceptedtokens.ts

```diff
@@ -0,0 +1,45 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import config from '../src/config';
+
+export class addChangeAcceptedtokens1648066794387
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    const changeOrganization = (
+      await queryRunner.query(`SELECT * FROM organization
+        WHERE label = 'change'`)
+    )[0];
+
+    const mainnetNativeToken = (
+      await queryRunner.query(`
+        SELECT * FROM token
+        WHERE symbol='ETH' and "networkId"=1
+      `)
+    )[0];
+
+    // for both production and staging
+    await queryRunner.query(
+      `INSERT INTO organization_tokens_token ("tokenId","organizationId") VALUES
+        (${mainnetNativeToken.id}, ${changeOrganization.id})
+      ;`,
+    );
+
+    // add test token for any other env
+    if (config.get('NODE_ENV') !== 'production') {
```

Comment:
We have problem and that is we have set `NODE_ENV=produciton` in all of our environments (develop, staging, production)
so this part will never run on our servers ( we might add new variable in config for specifying the environment, because I don't know how much side effects would have changing NODE_ENV ins taging

Reply:
Thanks, we had another variable called Environment.